### DISCUSSION TITLE
Windows 8.1

### DISCUSSION THREAD

**Discussion by githottub** [Rem0o/FanControl.Releases#686] [Q&A] [answered]
FAQ says only for Windows 10/11. However, software websites such as MajorGeeks indicate compatability down to 7. 
I have a Windows 8.1 HTPC I'm not upgrading due to it being the last OS version with Windows Media Center. 
Therefore, can you please tell me what was the last version of FanControl to work with 8.1? Thank you!

**Comment by Rem0o** [comment, marked as answer]
I would say arround 63-64 when I added Windows theming,

**Reply by githottub** [reply]
Thank you! I don't have the skills/tools to compile the source code; is there anywhere to get a compiled version of the older releases?

**Reply by Rem0o** [reply]
Scroll down there: https://github.com/Rem0o/FanControl.Releases/releases

**Reply by githottub** [reply]
I looked there previously but when the only files I saw said Source Code, I didn't realize there were compiled executables inside. Thanks again!
